runbotics/runbotics-orchestrator-ui/src/src-landing/components/Layout/Layout.tsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 
 import Footer from '#src-landing/components/Footer';
 import Header from '#src-landing/components/Header';
+import { MAIN_CONTENT_ID } from '#src-landing/utils/utils';
 
 const Layout = ({ children }) => (
     <>
         <Header />
-        <main>{children}</main>
+        <main id={MAIN_CONTENT_ID} tabIndex={-1}>{children}</main>
```

Comment:
what tabindex -1 enables here?

Reply:
it enables to set the keyboard focus on the main content